plugins/app-backend/src/lib/config.test.ts

```diff
@@ -80,14 +95,19 @@ describe('injectEnvConfig', () => {
     expect(fsMock.writeFile).toHaveBeenCalledTimes(1);
     expect(fsMock.writeFile).toHaveBeenCalledWith(
       resolvePath(MOCK_DIR, 'main.js'),
-      '/*__APP_INJECTED_CONFIG_MARKER__*/"{\\"x\\":0}"/*__INJECTED_END__*/',
+      '/*__APP_INJECTED_CONFIG_MARKER__*/"[{\\"data\\":{\\"x\\":0},\\"context\\":\\"test\\"}]"/*__INJECTED_END__*/',
```

Comment:
Just a thought - the frontend isn't technically interested in the context right? It's in a sense an information leakage of directory structures from backend concerns that one may not necessarily want to have sent over.

Reply:
It is! It's for providing more informative error messages, and the paths are sanitized in that they're only relative paths from the repo root or just the filename itself